src/labels/CustomLabels.labels

```diff
@@ -308,6 +308,14 @@
         <shortDescription>AssistiveTextNewWindow</shortDescription>
         <value>Link opens in a new window</value>
     </labels>
+    <labels>
+        <fullName>AssistiveTextRecordIcon</fullName>
+        <categories>AlternativeText</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>AssistiveTextRecordIcon</shortDescription>
```

Comment:
The shortDescription for custom labels should include a description of the label and how it's use instead of being a duplicate of the label name.